docs/source/api.rst

```diff
@@ -0,0 +1,89 @@
+.. _api-chapter:
+
+##############
+API DI-Sensors
+##############
+
+============
+Requirements
+============
+
+Before you check the API for the DI-Sensors, please make sure you have the ``DI-Sensors`` package installed. You can do this by checking with ``pip`` by typing the following command.
```

Comment:
question:  will `pip` display it properly if it's been installed via the curl command?

Reply:
Yep, the `DI-Sensors` package will appear provided the install command has been run.